src/constants/index.js

```diff
@@ -0,0 +1,40 @@
+const GAME_VALUE = Object.freeze({
+  LOTTO_PRICE: 1_000,
+  LOTTO_SIZE: 6,
+
+  MAX_LOTTO_NUMBER: 45,
+  PRIZE: [2_000_000_000, 30_000_000, 1_500_000, 50_000, 5_000],
+});
+
+const FORMATTING_TYPE = Object.freeze({
+  BOUGHT_LOTTOS: 0,
+  WINNING_STATUS: 1,
+  PROFIT_RATE: 2,
```

Comment:
0, 1, 2 는 어떤 의미예요??

Reply:
`0`, `1`, `2` 자체는 큰 의미가 없습니다. `view/output.js` 에서 주어진 `formattingType`(어떤 기능을 수행할 것인지를 적는 곳) 에 따라 어떤 작업을 수행할지가 결정되는데, `'BOUGHT_LOTTOS'` 와 같이 문자열 그대로를 적거나, 의미를 알아보기 힘든 `0`, `1`, `2` 를 그대로 적는 것은 별로 좋지 않아 보여서, `0`, `1`, `2` 를 사용하되 상수로 의미를 넣는 방법을 채택했습니다.

기능을 알아보기 힘든 코드라면 변수 변경이나 주석 추가를 하는 것이 좋으려나요?

Reply:
아하! 의미있는 값들이 아니라면 보통 같은 문자열을 넣는 것 같아요! 나중에 typescript 적용하게 되면 type 이나 enum 을 적용하면 될거예요~!

```suggestion
  BOUGHT_LOTTOS: "BOUGHT_LOTTOS",
  WINNING_STATUS: "WINNING_STATUS",
  PROFIT_RATE: "PROFIT_RATE",
```